tests/performance/density.go

```diff
@@ -196,3 +192,21 @@ func waitRunningVMI(virtClient kubecli.KubevirtClient, vmiCount int, timeout tim
 		return running
 	}, timeout, 10*time.Second).Should(Equal(vmiCount))
 }
+
+func withMemoryLimit(value string) libvmi.Option {
+	return func(vmi *kvv1.VirtualMachineInstance) {
+		if vmi.Spec.Domain.Resources.Limits == nil {
+			vmi.Spec.Domain.Resources.Limits = k8sv1.ResourceList{}
+		}
+		vmi.Spec.Domain.Resources.Limits[k8sv1.ResourceMemory] = resource.MustParse(value)
+	}
+}
+
+func withCpuLimit(value string) libvmi.Option {
+	return func(vmi *kvv1.VirtualMachineInstance) {
+		if vmi.Spec.Domain.Resources.Limits == nil {
+			vmi.Spec.Domain.Resources.Limits = k8sv1.ResourceList{}
+		}
+		vmi.Spec.Domain.Resources.Limits[k8sv1.ResourceCPU] = resource.MustParse(value)
+	}
+}
```

Comment:
These two seem to be very basic and without any special logic. I think they can be placed under `libvmi`.

Reply:
done